src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -314,18 +313,18 @@ const SparkLineLoadingBubble = styled(LongLoadingBubble)`
   height: 4px;
 `
 
-const L2NetworkLogo = styled.div<{ networkUrl?: string }>`
-  height: 12px;
-  width: 12px;
+export const L2NetworkLogo = styled.div<{ networkUrl?: string; size?: string }>`
+  height: ${({ size }) => size ?? '12px'};
```

Comment:
just to double check - this is the only way to do a default value in style components? ideally we could set the fallback value to size to be 12 and then you don't have to have these ternary expressions

Reply:
It looks like my only alternative is https://styled-components.com/docs/advanced, but i don't see any other usage of defaultProps in our repo

Reply:
repetitive ternarys in styled components are really annoying